tests/api/test_session_view_service.py

```
@@ -719,6 +723,47 @@ async def get_response(self, messages, **kwargs):
+        Message(
+            id="msg_user",
+            role="user",
+            content="howdy",
+            category=MessageCategory.DIALOG,
+            timestamp="2026-02-03T00:00:00",
+        )
+    )
+    core = _core([session])
+
+    class _UnexpectedAPIClient:
+        def __init__(self, model_config):
```

Ruff (0.15.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)